### PR TITLE
feat: Adds schema/android implementation for Bind attributes of Text widget

### PR DESCRIPTION
**Pull request by hernandazevedozup**
Adds schema/android implementation for Bind attributes of Text widget

https://github.com/ZupIT/beagle/issues/283

### CODE REVIEW

File: android/beagle/src/main/java/br/com/zup/beagle/android/utils/StyleManager.kt [hunk 1]
```diff
@@ -46,7 +46,11 @@ class StyleManager(
             view.applyViewBackgroundAndCorner(Color.TRANSPARENT, component)
         } else when (component) {
             is Text -> {
-                applyStyleId(context, component.styleId ?: "", view, component)
+                if (component.styleId == null || component.styleId is Bind.Value) {
+                    applyStyleId(context, (component.styleId?.value ?: "") as String, view, component)
```
Comment (LG95) [inline android/beagle/src/main/java/br/com/zup/beagle/android/utils/StyleManager.kt:50]: I'm not sure, but this should work if Moshi is specializing the values.
```suggestion
                if (component.styleId == null || component.styleId is Bind.Value<String>) {
                    applyStyleId(context, (component.styleId?.value ?: ""), view, component)
```

**Comment by tuliopereirazup** [comment]
Does iOS is doing the same thing? I mean, create a PR for each component.

**Comment by hernandazevedozup** [comment]
> Does iOS is doing the same thing? I mean, creating a PR for each component.

@victormr

File: android/beagle/src/main/java/br/com/zup/beagle/android/utils/StyleManager.kt [hunk 2]
```diff
@@ -46,7 +46,11 @@ class StyleManager(
             view.applyViewBackgroundAndCorner(Color.TRANSPARENT, component)
         } else when (component) {
             is Text -> {
-                applyStyleId(context, component.styleId ?: "", view, component)
+                if (component.styleId == null || component.styleId is Bind.Value<String>) {
+                    applyStyleId(context, (component.styleId?.value ?: ""), view, component)
```
Comment (LG95) [inline android/beagle/src/main/java/br/com/zup/beagle/android/utils/StyleManager.kt:50]:
```suggestion
                if (component.styleId == null || component.styleId is Bind.Value) {
                    applyStyleId(context, (component.styleId?.value ?: "") as String, view, component)
```

**Pull request merged**